Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -109,13 +111,13 @@ public struct TargetSourcesBuilder {
         
         // Check if paths that are enumerated in targets: [] exist
         guard self.fs.exists(at) else {
-            return StringError("\(at) could not be found")
+            return StringError("File not found")
         }
 
         // Excludes, Sources, and Resources should be found at the root of the package and or
         // its subdirectories
         guard at.pathString.hasPrefix(cwd.pathString) else {
-            return StringError("\(at) should be found within the root of \(cwd)")
+            return StringError("The current working directory '\(cwd)' should contain: ")
```

Comment:
It seems wrong for this code to be looking at the current working directory.  It seems to me that this code should be looking at the `packagePath` or maybe the `targetPath` as the root directory of interest (depending on context), but not on external state like the process's current working directory.

Reply:
Right, I missed this, we should be looking at `packagePath` here, I think. 

Might also make sense to add an additional test with a custom package path to avoid making the assumption that the package is in the current working directory in the future.